perf/aiostress.py

```diff
@@ -20,12 +19,10 @@ def setUp(self):
         Source:
          https://oss.oracle.com/~mason/aio-stress/aio-stress.c
         """
-        aiostress_c = self.params.get('aiostress_c', default='aio-stress.c')
-        c_path = self.get_data_path(aiostress_c)
-        shutil.copy(c_path, self.srcdir)
+        aiostress = self.fetch_asset('https://oss.oracle.com/~mason/aio-stress/aio-stress.c')
         os.chdir(self.srcdir)
         # This requires libaio.h in order to build
```

Comment:
Unrelated to this PR, but how about adding (I used first match so the url should be probably changed):

```python
libaio = self.fetch_asset('http://compbio.cs.toronto.edu/repos/snowflock/xen-3.0.3/tools/libaio/src/libaio.h')
...
process.run('gcc -Wall -laio -lpthread -I%s -o aio-stress %s' % (os.path.dirname(libaio), aiostress))
```